webdriver/tests/bidi/input/perform_actions/pointer_mouse.py

```diff
@@ -211,44 +211,75 @@ async def test_drag_and_drop(
 
 
 @pytest.mark.parametrize("drag_duration", [0, 300, 800])
-async def test_drag_and_drop_with_draggable_element(
-    bidi_session, top_context, get_element, load_static_test_page, drag_duration
-):
-    new_context = await bidi_session.browsing_context.create(type_hint="window")
-    await load_static_test_page(page="test_actions.html", context=new_context)
+async def test_drag_and_drop_with_draggable_element(bidi_session, top_context,
+                                                    get_element,
+                                                    load_static_test_page,
+                                                    drag_duration):
+    await load_static_test_page(page="test_actions.html")
+
     drag_target = await get_element("#draggable")
     drop_target = await get_element("#droppable")
 
     # Conclude chain with extra move to allow time for last queued
     # coordinate-update of drag_target and to test that drag_target is "dropped".
     actions = Actions()
-    (
-        actions.add_pointer()
-        .pointer_move(x=0, y=0, origin=get_element_origin(drag_target))
-        .pointer_down(button=0)
-        .pointer_move(
-            x=50, y=25, duration=drag_duration, origin=get_element_origin(drop_target)
-        )
-        .pointer_up(button=0)
-        .pointer_move(80, 50, duration=100, origin="pointer")
-    )
+    actions.add_pointer().pointer_move(
+        x=0, y=0, origin=get_element_origin(drag_target)).pointer_down(
+            button=0).pointer_move(
+                x=0,
+                y=0,
+                duration=drag_duration,
+                origin=get_element_origin(drop_target)).pointer_up(button=0)
```

Comment:
With those changes the individual action steps are very hard to read. Would it be fine with you to revert it by using surrounding brackets again and having each action on its own line?

Reply:
Done.